### PR TITLE
feat: Add option to disable tick sounds during break

### LINKED ISSUE #124: Mute tick sounds during break time
Is there an option to mute tick sounds during the break time, and have it chime only at the end of the break?

### PR DESCRIPTION
**Pull request by johnjacobkenny**
Closes #124 

Please let me know if any changes are needed. All feedback is appreciated. :slightly_smiling_face:

### CODE REVIEW

File: src/renderer/components/Audio.vue [hunk 1]
```diff
@@ -62,6 +68,8 @@ export default {
     EventBus.$on('timer-tick', () => {
       this.$refs['audio-tick'].volume = this.volume
       if (!this.tickSounds) return
```
Comment (Splode) [inline src/renderer/components/Audio.vue:70]: @johnjacobkenny this check on the `tickSounds` property will short-circuit the playing of tick sounds during breaks if `false`.

**Comment by Splode** [comment]
Thank you, @johnjacobkenny, for the contribution! I think it's a great addition.

However, there's an issue with the new feature. If the option _Tick Sounds - Work_ is unchecked, but _Tick Sounds - Break_ is checked, the tick sounds will not play.

File: src/renderer/components/Audio.vue (hunk 1 above)
Comment (johnjacobkenny) [reply src/renderer/components/Audio.vue]: Yes!

**Comment by johnjacobkenny** [comment]
I have changed the logic and tested it. Please let me know if the logic looks good and maintainable.

**Comment by Splode** [comment]
@johnjacobkenny looks good, thank you for the contribution!

**Pull request merged**